hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneListStatusHelper.java

```diff
@@ -340,7 +340,7 @@ public OzoneFileStatus getStatus(String prefixPath, long scmBlockSize,
             String prefixKey, String startKey) throws IOException {
       this.iterType = iterType;
       this.table = table;
-      this.tableIterator = table.iterator();
+      this.tableIterator = table.iterator(prefixKey);
```

Comment:
Does this mean we no longer need the path in the next couple lines.
```
      if (!StringUtils.isBlank(prefixKey)) {
        tableIterator.seek(prefixKey);
      }
```

Reply:
+1. These seem unnecessary now.

Reply:
Thanks @duongkame